Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -158,16 +158,17 @@ Breaking Change Rules
 * Adding `virtual` to a member  
 > Make note, that marking a member virtual might cause previous consumers to still call the member non-virtually.
 
-* Introducing or removing an override
-> Make note, that introducing an override might cause previous consumers to skip over the override when calling `base`.
-
 &#10007; **Disallowed**
 * Adding an member to an interface
 
 * Adding an abstract member to a type when there _are accessible_ (`public` or `protected`) constructors and the type is not `sealed`
 
 * Adding a constructor to a class which previously had no constructor, without also adding the default constructor
 
+* Introducing or removing an override to non-sealed `protected` or `protected internal` types
```

Comment:
> Should this refer to methods instead of types?

All overridable members - methods and properties - of that accessible outside of the declaring assembly - public, protected, internal.

> Also, why does `protected` or `protected` internal matter here? It should apply to all methods visible in the public surface.

Actually, these are the only ones that matter. To C#, at least. Any reference from a non derived class to these members will be a virtual call (`callvirt`) to the definition of the member. Only on a derived class, an invocation to `base.Member` is a static call (`call`) to the nearest know implementation in the inheritance chain.